packages/react-native/src/primitives/TextField/TextField.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useMemo } from 'react';
+import { TextInput, View, ViewStyle } from 'react-native';
+import { Label } from '../Label';
+
+import { styles } from './styles';
+import { TextFieldProps } from './types';
+
+export default function TextField({
+  accessibilityLabel,
+  accessibilityRole,
+  accessibilityState,
+  accessible = true,
+  containerStyle,
```

Comment:
I just realized that for most of our primitives, we're passing the `style` prop to the outer container. However, I also like the specificity of naming it `containerStyle`. Maybe we should align on one of these approaches?

Reply:
Not sure I follow, we pass in specific style props on all our primitives.

Reply:
Synced offline, will allign when adding theming, w/ styles for different states and variants.